api/src/main/java/org/pmiops/workbench/genomics/WgsCohortExtractionService.java

```diff
@@ -75,7 +80,11 @@ public TerraJob submitGenomicsCohortExtractionJob(DbWorkspace workspace, Long co
     WorkbenchConfig.WgsCohortExtractionConfig cohortExtractionConfig =
         workbenchConfigProvider.get().wgsCohortExtraction;
 
+    FirecloudWorkspace fcWorkspace = fireCloudService.getWorkspace(workspace).get().getWorkspace();
```

Comment:
I would make these variable names clearer, since there are multiple workspaces involved, e.g. `userWorkspace` and `fcUserWorkspace`